editor/src/clj/editor/ui.clj

```diff
@@ -1914,10 +1914,17 @@
   [^Scene scene evaluation-context]
   (let [visible-command-contexts (visible-command-contexts scene)
         current-command-contexts (current-command-contexts scene)
-        root (.getRoot scene)]
+        root (.getRoot scene)
+        app-view (-> current-command-contexts first :env :app-view)
+        active-tab (g/maybe-node-value app-view :active-tab evaluation-context)]
     (doseq [td (vals (user-data root ::toolbars))]
-      (refresh-toolbar td visible-command-contexts evaluation-context)
-      (refresh-toolbar-state (:control td) current-command-contexts evaluation-context))))
+      (let [control (:control td)
+            is-active (boolean (when active-tab
+                                 (nodes-along-path? control (.getContent ^Tab active-tab) root)))]
```

Comment:
This would also hide all toolbars when we focus out of tabs. I don't think that's bad, since we already do that when we focus on the asset tree.